### PR TITLE
Fix SummEval NaN scores

### PR DESCRIPTION
**Pull request by Muennighoff**
If all scores are the same in SummEval, the correlation computation will produce NaN scores.
For example for the below Machine translations, some models encode them as the same embedding.
I couldn't come up with a better solution than skipping - any better ideas?

```python
["japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers in the air next to their daughters . images are from the 37-year-old 's latest book that roughly translates to daughter and a salaryman each image . the images are also to be part of a book of the father-daughter photographer . the 37-year-old has been captured by photographer photographer yûki .", "japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers pictured next to their fathers leaping into the air . in each picture the daughter looks directly into the camera smiling while her father pulls a dramatic pose . the images are from the 37-year-old 's latest book which roughly translates into daughter and salary man .", "japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers pictured next to their fathers leaping into the air . the images are from the 37-year-old 's latest book which roughly translates into daughter and salary man . a series of images by photographer yûki aoyama sees fathers leaping into the air next to their daughters .", "photographer yûki aoyama 's latest series of images capture po-faced teenagers pictured next to their fathers leaping into the air . in each picture the daughter looks directly into the camera smiling while her father pulls a dramatic pose .", "japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers pictured next to their fathers leaping into the air . in each picture the daughter looks directly into the camera smiling while her father pulls a dramatic pose .", "japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers pictured next to their fathers leaping into the air . in each picture the daughter looks directly into the camera smiling while her father pulls a dramatic pose .", "japanese photographer yûki aoyama 's latest series of images capture po - faced teenagers pictured next to their fathers leaping into the air . in each picture the daughter looks directly into the camera smiling while her father pulls a dramatic pose . a series of images by photographer yûki aoyama sees fathers leaping into the air next to their daughters the images are from the 37 - year - old 's latest book which roughly translates into daughter and salary man .", "japanese photographer yûki aoyama's latest series of images capture po-faced teenagers pictured next to their fathers . in each picture the daughter looks directly into the camera smiling while her father pulls a dramatic pose . the images are said to be part of a book which roughly translates as daughter and salaryman .", 'A father-daughter pair who have been posing for their father in front of various Japanese landmarks are making a name for themselves online. Related:', "japanese photographer y ? ki aoyama's latest series of images capture po-faced teenagers . in each picture the daughter looks directly into the camera smiling while her father pulls a dramatic pose . the images are said to be part of a book which roughly translates as daughter and salary man .", "japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers pictured next to their fathers leaping into the air . a series of images by photographer yûki aoyama sees fathers leaping into the air next to their daughters . the images are from the 37-year-old 's latest book which roughly translates into daughter and salary man .", "sick of awkward father-daughter portraits ? well one photographer has found an effective - if a little odd - way of making them more interesting . japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers pictured next to their fathers leaping into the air .", "Japanese photographer Yûki Aoyama 's latest series of images capture po-faced teenagers pictured next to their fathers leaping into the air . In each picture the daughter looks directly into the camera smiling while her father pulls a dramatic pose . The images are said to be part of a book which roughly translates into Daughter and Salaryman .", "japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers pictured next to their fathers . the images are from the 37-year-old 's latest book which roughly translates as daughter and salaryman . each image sees the daughter stood po-faced as their father makes an energetic leap .", "japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers next to their daughters . images are said to be part of a book which roughly translates into daughter and salary man . the 37-year-old 's images include .", "japanese photographer yûki aoyama 's latest series of images capture po-faced teenagers . images are from the 37-year-old 's latest book which translates into daughter and salary man . photographer yûki aoyama sees fathers leaping into air next to their daughters . in each picture the daughter looks directly into the camera smiling while father pulls a dramatic pose ."]
```

### CODE REVIEW

File: mteb/evaluation/evaluators/SummarizationEvaluator.py
Comment (NouamaneTazi) [inline mteb/evaluation/evaluators/SummarizationEvaluator.py:65]: I think it's better to use the local logger instead of the global one 
```suggestion
                logger.info(f"Skipping sample {i} due to equal scores")
```

**Comment by NouamaneTazi** [comment]
LGTM! Would be cool to get @nreimers's opinion also before merging :)

**Review by NouamaneTazi — APPROVED** [review]

**Comment by Muennighoff** [comment]
@nreimers Okay with you if we merge this? Curious if you have a better idea to treat this kind of behavior though

**Pull request merged**
